zerver/lib/markdown/preprocessor_priorities.py

```diff
@@ -10,9 +11,12 @@
     "generate_return_values": 510,
     "generate_api_arguments": 505,
     "include": 500,
+    # "include_wrapper": 500,
     "help_relative_links": 475,
     "setting": 450,
+    # "html_block": 20,
     "fenced_code_block": 25,
+    # "normalize_whitespace": 30,
```

Comment:
What is this sorted by?

Reply:
It is sorted by the numeric priority of the preprocessors.

Reply:
@andersk You're right, I did not pay enough attention to the order of the lines when adding them. I'll fix this shortly, thanks for pointing that out!